src/App.js

```diff
@@ -1,18 +1,28 @@
 import React from 'react';
 
+var todoList = [
+  {
+    id: 1,
+    title: "Wake up"
+  },
+  {
+    id: 2,
+    title: "Take a shower"
+  },
+  {
+    id: 3,
+    title: "Walk the dog"
+  }
+];
+
 function App() {
-  return (
-    <div style={{ textAlign: 'center' }}>
-      <header>
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+  return(
+  <ul>
+    {todoList.map( (item) =>
+       <li key={item.id}>{item.title}</li>
+    )}
```

Comment:
Great variable naming! You always want your variables to be as descriptive and specific as possible.